### ISSUE TITLE
Some alfa-wcag tests does not follow the path-convention

### ISSUE THREAD

**Issue by NiclasHedam** [Siteimprove/alfa#25]
The source file `alfa-wcag/src/title/rule.ts` is tested by `alfa-wcag/test/title.spec.tsx`.
The test file should have had the path `alfa-wcag/test/title/rule.spec.tsx`.

**Comment by kasperisager** [comment]
Good point!
